.github/workflows/ci.yml

```diff
@@ -36,15 +36,15 @@ jobs:
         run: |
           poetry run pre-commit run --all-files
 
+      - name: Use .env.example
```

Comment:
Not sure this does anything? We aren't telling the make build command to use the env file, so this is redundant